mm2src/mm2_main/src/lp_native_dex.rs

```diff
@@ -499,15 +541,6 @@ pub async fn lp_init_continue(ctx: MmArc) -> MmInitResult<()> {
 pub async fn lp_init(ctx: MmArc, version: String, datetime: String) -> MmInitResult<()> {
     info!("Version: {} DT {}", version, datetime);
 
-    #[cfg(not(target_arch = "wasm32"))]
-    {
-        let dbdir = ctx.dbdir();
-        fs::create_dir_all(&dbdir).map_to_mm(|e| MmInitError::ErrorCreatingDbDir {
-            path: dbdir.clone(),
-            error: e.to_string(),
-        })?;
-    }
-
```

Comment:
I think the base db dir should be created here to allow the below to work https://github.com/KomodoPlatform/komodo-defi-framework/blob/ffe761ee89768ad773dcb3d963783c9fbbf1caba/mm2src/mm2_main/src/lp_wallet/mnemonics_storage.rs#L22-L32 which is run as part of this function https://github.com/KomodoPlatform/komodo-defi-framework/blob/a78704dec81b487d820ddab5f61d1b0e7e86154e/mm2src/mm2_main/src/lp_native_dex.rs#L544-L545 and saving mnmonic file will fail otherwise.

Reply:
thanks for the catch !